toolkit/templates/framework-notice.html

```diff
@@ -0,0 +1,13 @@
+<aside role="complementary" aria-labelledby="framework-notice-heading" class="framework-notice">
+  <h2 id="framework-notice-heading">
+    {{ heading }}
+  </h2>
+  {% if subheading %}
+    <p class="framework-notice-subheading hint">
```

Comment:
Can this just have one class and put `colour: $secondary-text-color` in the SASS instead?